### PR TITLE
Sponsor update

### PR DESCRIPTION
**Pull request by Marcpezin**
* Adding OpenIt to the list of sponsors
* Adding "joining us" information

### CODE REVIEW

File: sponsors.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ comments: false
 ---
 XCP-ng sponsors span all industry sectors. They are trusting open source technologies to help them in their everyday business.
 
+> Want to join our sponsors? [Contact us](mailto:sales@vates.fr)
```
Comment (olivierlambert) [inline sponsors.md]: we should use "contact@xcp-ng.org" email address instead

**Review by olivierlambert — CHANGES_REQUESTED** [review]

**Comment by Marcpezin** [comment]
done

File: sponsors.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ comments: false
 ---
 XCP-ng sponsors span all industry sectors. They are trusting open source technologies to help them in their everyday business.
 
-> Want to join our sponsors? [Contact us](mailto:sales@vates.fr)
+> Want to join our sponsors? [Contact us](mailto:contact@xcp-ng.org)sta
```
Comment (olivierlambert) [inline sponsors.md]: sta?

**Review by olivierlambert — CHANGES_REQUESTED** [review]

File: sponsors.md (hunk 2 above)
Comment (Marcpezin) [reply sponsors.md]: damn, corrected.

**Review by olivierlambert — APPROVED** [review]

**Pull request merged**